File: src/main/java/ru/dragonestia/apocalypse/item/ClothItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class ClothItem extends Item implements AdvItem {

    public ClothItem(Integer meta, int count) {
        super(ApocalypseID.CLOTH, meta, count, "Cloth");
    }

    @Override
    public String getRuName() {
        return "Ткань";
    }

    @Override
    public String getIcon() {
        return "textures/items/cloth";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/ClothItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class ClothItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.CLOTH, 0, count, "Cloth"), AdvItem {

    override fun getRuName(): String = "Ткань"

    override fun getIcon(): String = "textures/items/cloth"

}